src/Mix.h

```diff
@@ -115,35 +115,34 @@ class AUDACITY_DLL_API Mixer {
    // Processing
    //
 
-   /// Process a maximum of 'maxSamples' samples and put them into
-   /// a buffer which can be retrieved by calling GetBuffer().
-   /// Returns number of output samples, or 0, if there are no
-   /// more samples that must be processed.
+   //! Process a maximum of 'maxSamples' samples and put them into the buffer, at GetBuffer().
+   /*! @return number of output samples, or 0, if there are no more samples that must be processed. */
    size_t Process(size_t maxSamples);
 
-   /// Restart processing at beginning of buffer next time
-   /// Process() is called.
+   //! Restart processing at beginning of buffer next time Process() is called.
    void Restart();
 
-   /// Reposition processing to absolute time next time
-   /// Process() is called.
+   //! Reposition processing to absolute time next time Process() is called.
    void Reposition(double t, bool bSkipping = false);
 
-   // Used in scrubbing.
+   //! Used in scrubbing.
```

Comment:
This probably will have an undesired effect. "Used in scrubbing" will be used as documentation for SetTimesAndSpeed

Reply:
That is what I intended.

Reply:
So it doesn't apply to the functions below?